index.qmd

```diff
@@ -84,48 +84,102 @@ accessibility tests, and more, consult our [Guides](#guides)
 
 ### Installation
 
-To install the workbench, make sure you have a working version of R and 
-pandoc/RStudio installed (see [the workbench setup instructions for 
+:::: {.panel-tabset}
+
+#### Within R
+
+To install the workbench, make sure you have a working version of R and
+pandoc/RStudio installed (see [the workbench setup instructions for
 details](https://carpentries.github.io/sandpaper-docs/)).
 
-::: {.callout-tip collapse='true'}
+From there, you can install the workbench packages and their dependencies from
+our [Carpentries R Universe](https://carpentries.r-universe.dev) inside of R:
+
+```r
+install.packages(c("sandpaper", "varnish", "pegboard", "tinkr"),
+  repos = c("https://carpentries.r-universe.dev/", getOption("repos")))
+```
+
+#### From conda-forge
+
+All of The Workbench tools, and their dependencies, are packaged and distributed
+as built binaries in the form of conda packages on [conda-forge](https://conda-forge.org/)
+for Linux, macOS, and Windows.
+You can setup an environment for lesson development that includes all dependencies
+(including R) with the following.
 
-#### Setup with Anaconda
+::: {.panel-tabset}
+
+##### With Pixi
 
-Thanks to Travis Wrightsman for [providing the instructions to setup for
-anaconda](https://github.com/carpentries/workbench/issues/50). These lines
-will set up an anaconda environment that includes all the dependencies you need
-to use The Workbench. 
+[Pixi](https://pixi.sh/) environments are fully reproducible by default.
+From the project directory, initialize a Pixi workspace
 
 ```sh
-conda create -n workbench 'git>=2.28' 'r-base>=3.6' 'pandoc>=2.11' pkg-config libxslt
-conda activate workbench
-R -e 'install.packages(c("sandpaper", "varnish", "pegboard", "tinkr"), \
-  repos = list(carpentries="https://carpentries.r-universe.dev/", CRAN="https://cloud.r-project.org"))'
+pixi init
 ```
 
-:::
+and then add Git and The Workbench tool packages (`r-pegboard` is a dependency of `r-sandpaper`)
 
-From there, you can install the workbench packages and their dependencies from
-our [Carpentries R Universe](https://carpentries.r-universe.dev) inside of R:
+```sh
+pixi add git r-varnish r-sandpaper
```

Comment:
```suggestion
pixi add git r-tinkr r-varnish r-pegboard r-sandpaper
```

Reply:
Thanks, but I'd like to avoid this given https://github.com/carpentries/workbench/pull/90/files#r2075980099.